### DISCUSSION TITLE
Please update GSAP

### DISCUSSION THREAD

**Discussion by joe-mohan** [hypernym-studio/nuxt-gsap#60] [General]
possible to bump gsap to v3.12.5? We're running into this issue:

[https://gsap.com/community/forums/topic/39323-scrolltrigger-matchmedia-nextjs-works-and-stops-on-resize/](https://gsap.com/community/forums/topic/39323-scrolltrigger-matchmedia-nextjs-works-and-stops-on-resize/)

All the best!

**Comment by ivodolenc** [comment]
Hi, for some reason I didn't get a notification of this discussion, I need to review the notification settings since this is from `organization` repo.

Updated to the [`latest`](https://github.com/hypernym-studio/nuxt-gsap/releases/tag/v2.4.3) version.
